src/ontology/components/cob-to-external.owl

```diff
@@ -1,18 +1,44 @@
 <?xml version="1.0"?>
 <rdf:RDF xmlns="http://purl.obolibrary.org/obo/cob/components/cob-to-external.owl#"
      xml:base="http://purl.obolibrary.org/obo/cob/components/cob-to-external.owl"
+     xmlns:CL="http://purl.obolibrary.org/obo/CL_"
+     xmlns:GO="http://purl.obolibrary.org/obo/GO_"
+     xmlns:PO="http://purl.obolibrary.org/obo/PO_"
+     xmlns:PR="http://purl.obolibrary.org/obo/PR_"
+     xmlns:RO="http://purl.obolibrary.org/obo/RO_"
+     xmlns:SO="http://purl.obolibrary.org/obo/SO_"
+     xmlns:BFO="http://purl.obolibrary.org/obo/BFO_"
+     xmlns:COB="http://purl.obolibrary.org/obo/COB_"
+     xmlns:GEO="http://purl.obolibrary.org/obo/GEO_"
+     xmlns:IAO="http://purl.obolibrary.org/obo/IAO_"
+     xmlns:MOP="http://purl.obolibrary.org/obo/MOP_"
+     xmlns:OBI="http://purl.obolibrary.org/obo/OBI_"
+     xmlns:PCO="http://purl.obolibrary.org/obo/PCO_"
+     xmlns:XAO="http://purl.obolibrary.org/obo/XAO_"
+     xmlns:ZFA="http://purl.obolibrary.org/obo/ZFA_"
```

Comment:
No idea how this happened but it must have happened because of 

https://github.com/ontodev/robot/pull/1106

@balhoff can you get a sense if that is... intended/correct?

Reply:
That would be my guess, if these are defined as prefixes in an upstream ontology.

Reply:
These won't actually get used for elements in RDF/XML though, since you can't have local names in XML that start with a digit. Defining them doesn't hurt though.